includes/class.llms.generator.php

```diff
@@ -162,43 +162,28 @@ public function generate() {
 	 */
 	protected function get_error_code( $code, $class ) {
 
-		// PHP Warnings, notices, exceptions.
-		switch ( $code ) {
-			case E_ERROR: // 1.
-				return 'E_ERROR';
-			case E_WARNING: // 2.
-				return 'E_WARNING';
-			case E_PARSE: // 4.
-				return 'E_PARSE';
-			case E_NOTICE: // 8.
-				return 'E_NOTICE';
-			case E_CORE_ERROR: // 16.
-				return 'E_CORE_ERROR';
-			case E_CORE_WARNING: // 32.
-				return 'E_CORE_WARNING';
-			case E_COMPILE_ERROR: // 64.
-				return 'E_COMPILE_ERROR';
-			case E_COMPILE_WARNING: // 128.
-				return 'E_COMPILE_WARNING';
-			case E_USER_ERROR: // 256.
-				return 'E_USER_ERROR';
-			case E_USER_WARNING: // 512.
-				return 'E_USER_WARNING';
-			case E_USER_NOTICE: // 1024.
-				return 'E_USER_NOTICE';
-			case E_STRICT: // 2048.
-				return 'E_STRICT';
-			case E_RECOVERABLE_ERROR: // 4096.
-				return 'E_RECOVERABLE_ERROR';
-			case E_DEPRECATED: // 8192.
-				return 'E_DEPRECATED';
-			case E_USER_DEPRECATED: // 16384.
-				return 'E_USER_DEPRECATED';
+		// See if the error code is a native php exception code constant.
+		$ret = llms_error_constant_to_code( $code );
```

Comment:
shall we use a more meaningful name here so that it's more clear we only map native PHP error constants?!

Reply:
you mean a more meaningful function name? what do you propose?

Reply:
- [ ] `llms_php_native_error_constant_to_code`
- [ ] `llms_native_error_constant_to_code`

Reply:
How about just `llms_php_error_constant_to_code` ?

Reply:
(Yours are fine if you're not into the whole brevity thing)

Reply:
ok, fine enough <3